CHANGELOG.md

```diff
@@ -42,13 +43,15 @@ different versioning scheme, following the Haskell community's
 
 ## C# ###
 
+* Added gRPC integration
 * Added controls to cap incremental allocation between reads in
   `Bond.IO.Unsafe.InputStream`.
 * Extended bug parsing JSON when a string value is a date.
   [Pull request #358](https://github.com/Microsoft/bond/pull/358)
 
 ### C# Comm ###
 
+* C# Comm now deprecated, use Bond-over-gRPC.
```

Comment:
Add a note that you now have to pass `--comm` when doing codegen to get the Comm files. Consider linking to a .csproj where this happens.